src/components/exercise-generation/exercise-generation-wizard/difficulty.tsx

```diff
@@ -76,3 +77,16 @@ export function Difficulty({
     </div>
   )
 }
+
+// No i18n needed for now, as the exercise generation only works for German
+export function mapDifficultyToDescriptor(
+  difficulty: ExerciseGenerationDifficulty
+): string {
+  const mapping: { [key in ExerciseGenerationDifficulty]: string } = {
+    low: 'einfachen',
+    medium: 'typischen',
+    high: 'komplexen',
+  }
+
+  return mapping[difficulty]
+}
```

Comment:
note:
this seems a bit complicated, maybe something like this is enough (and you could probably remove the type definition above as well)?
``` ts
const difficultyToDescriptor = {
  low: 'einfachen',
  medium: 'typischen',
  high: 'komplexen',
} as const 
```
and accessing it with `difficultyToDescriptor[difficulty]` directly.